backend/app/routers/metadata.py

```diff
@@ -70,6 +71,49 @@ async def get_metadata_definition_list(
     return page.dict()
 
 
+@router.put(
+    "/definition/{metadata_definition_id}", response_model=MetadataDefinitionOut
+)
+async def update_metadata_definition(
+    metadata_definition: MetadataDefinitionIn,
+    metadata_definition_id: str,
+    user=Depends(get_current_user),
+):
+    existing_count = await MetadataDefinitionDB.find(
+        {
+            "$and": [
+                {"_id": {"$ne": PydanticObjectId(metadata_definition_id)}},
```

Comment:
why would "$ne" needed?

Reply:
Want to find out other metadata definitions using the same name.